core/file_api.php

```diff
@@ -69,8 +69,38 @@ function file_attach_files( $p_bug_id, $p_files, $p_bugnote_id = 0 ) {
 	}
 
 	$t_file_infos = array();
+	$t_zip_attachments = config_get_global( 'zip_attachments' );
+	if ( ON == $t_zip_attachments && extension_loaded( 'zip' ) ) {
+		$t_zip_attachments_excluded_extensions = array_map( 'strtolower', config_get_global( 'zip_attachments_excluded_extensions' ) );
+		$t_zip__attachments_minimum_size = config_get_global( 'zip_attachments_minimum_size' ) ;
```

Comment:
Can't work, variable name is `$t_zip__attachments_minimum_size` but must be `$t_zip_attachments_minimum_size`